modules/forum/src/main/ForumDelete.scala

```diff
@@ -19,16 +19,15 @@ final class ForumDelete(
       .allByUserCursor(user)
       .documentSource()
       .mapAsyncUnordered(4): post =>
-        postApi.viewOf(post).flatMap { _.so(deletePost) }
+        postApi.viewOf(post).flatMap(_.so(deletePost))
       .runWith(Sink.ignore)
       .void
 
   def deleteTopic(view: PostView)(using Me): Funit =
     for
-      postIds <- postRepo.idsByTopicId(view.topic.id)
```

Comment:
I removed it because We didn't use `postIds` but this probably caused by an oversight: https://github.com/lichess-org/lila/commit/5c94dc2a9c8988ab59e04882ca9db615415833fc#diff-27d0d9308aaabb53c6b0b2d6f43db2c751e7a7577ec4041e031d8d6f3338469e